src/content/en/updates/2017/04/headless-chrome.md

```diff
@@ -335,13 +343,40 @@ Demos
 
 ## FAQ
```

Comment:
I think FAQs are an OK stopgap for quickly updating an article, but I can see Joe's point to integrate it into the relevant sections. I'll try to give examples of how to integrate into the doc.